src/python/tools/Report_NonAlterableUrls_FindHttpURLs.csv

```diff
@@ -0,0 +1,97 @@
+url,relativeFilepath
```

Comment:
double check manually that the urls here are really non alterable. And update your script to handle valid cases